tests/unit/CliftTypes.cpp

```diff
@@ -0,0 +1,51 @@
+//
+// This file is distributed under the MIT License. See LICENSE.md for details.
+//
+
+#include "mlir/IR/Diagnostics.h"
+
+#include "revng-c/mlir/Dialect/Clift/IR/Clift.h"
+#include "revng-c/mlir/Dialect/Clift/IR/ImportModel.h"
+
+#include "ModelTypes.inc"
```

Comment:
What's with the `.inc` filename? I don't thing we ever use them anywhere else. Just make it a regular `.h` please.

Reply:
See here: https://rev.ng/gitlab/revng-private/revng-c/-/merge_requests/193#note_23454

Reply:
Thanks for pointing to the relevant conversation. It makes sense.